### ISSUE TITLE
Renaming in SparseStochMat module and others

### ISSUE THREAD

**Issue by j-i-l** [alexbovet/flow_stability#42]
The convention is (`scipy` is actually a terrible example for this):

- `snake_case` for module names
- `PascalCase` for class names

so:

SparseStochMat.py -> sparse_stoch_mat.py
`class sparse_stoch_mat` -> `class SparseStochMat` (I'd even go for just `SSM`)

There are other namings that we might want to change:

- sub-module `SynthTempNetwork.py` > `synth_temp_network.py`

And some we might want to keep (like `Upper_case` names for matrix-like objects).

**Comment by j-i-l** [comment]
@alexbovet should we go for that or keep the current naming scheme?

**Comment by alexbovet** [comment]
Yes, let's follow the convention (but keep SparseStochMat, not SSM, and keep upper case for matrices).
